packages/@react-spectrum/list/chromatic/ListView.chromatic.tsx

```diff
@@ -10,55 +10,124 @@
  * governing permissions and limitations under the License.
  */
 
-import {ActionButton} from '@react-spectrum/button';
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import {ActionMenu} from '@react-spectrum/menu';
+import Add from '@spectrum-icons/workflow/Add';
 import {Content, View} from '@react-spectrum/view';
-import {Flex} from '@react-spectrum/layout';
-import {Heading} from '@react-spectrum/text';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Folder from '@spectrum-icons/workflow/Folder';
+import {generatePowerset} from '@react-spectrum/story-utils';
+import {Grid, repeat} from '@react-spectrum/layout';
+import {Heading, Text} from '@react-spectrum/text';
 import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import Info from '@spectrum-icons/workflow/Info';
 import {Item, ListView} from '../';
+import {Link} from '@react-spectrum/link';
 import {Meta, Story} from '@storybook/react';
 import React from 'react';
 
-let flatOptions = [
-  {name: 'row 1'},
-  {name: 'row 2'},
-  {name: 'row 3'}
+// TODO: add overflow mode and thumbnails when those PRs are merged
+let states = [
+  {isQuiet: true},
+  // No visual difference between single and multiple
```

Comment:
Shouldn't we have a story with `selectedKeys` and then we'd need multiple selection to see how that looks?

Reply:
The story already has selectedKeys with multiple selected, though this should probably be `{selectionMode: 'multiple'},` rather than 'single`. Unlike TableView where `single` and `multiple` will either hide/display the select all checkbox, ListView doesn't have this distinction and thus we don't need to have both selection modes in the power set